src/js/dataMapping/awardsv2/additionalDetailsFinancialAssistance.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * additionalDetailsFinancialAssistance.js
+ * Created by Jonathan Hill 09/19/19
+ */
+
+
+const additionalDetailsContracts = () => {};
+
+export default additionalDetailsContracts;
```

Comment:
I think we can just delete this file?

Reply:
This was just a place holder for the following PR for financial assistance additional details which i apparently named the file right but not func.

I one file or three does not matter to me. I see benefits both ways.

Reply:
Cool. I'm good with either way too.